CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to Rust IPFS
+
+Welcome, and thank you for your interest in contributing to Rust IPFS. Issues and pull requests are encouraged.
+
+## Target Build
+
+Rust IPFS will always target the current _stable_ version of Rust that is released. Our CI/CD tests will reflect this.
+
+TODO: Instructions on installing the up-to-date stable toolchain
```

Comment:
The book discusses this at length, I think in general it'd best to always refer it: https://doc.rust-lang.org/book/ch01-01-installation.html. Now that I finally read this chapther I always link, I wished it discussed other options as well, but the current form is good enough.